src/test/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketClientMockUtils.java

```diff
@@ -159,6 +160,12 @@ private static BitbucketPullRequestValue getPullRequest() {
 
         pr.setSource(source);
 
+        BitbucketPullRequestValueDestination destination = new BitbucketPullRequestValueDestination();
+        branch = new BitbucketCloudBranch();
+        branch.setName("branch1");
+        destination.setBranch(branch);
+        pr.setDestination(destination);
```

Comment:
Missing call to `destination.setRepository(...)` - probably not used now in tests, but expected to be set anyway.